feature/calendar/src/commonMain/kotlin/com/ondot/calendar/contract/CalendarViewModel.kt

```
@@ -245,49 +237,49 @@ class CalendarViewModel(
 
-        launchResult(
-            block = {
                 scheduleRepository.toggleAlarm(
                     scheduleId = scheduleId,
                     isEnabled = enabled,
                 )
-            },
-            onError = {
+            } catch (e: CancellationException) {
+                throw e
+            } catch (t: Throwable) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)